packages/bpk-component-graphic-promotion/README.md

```diff
@@ -35,17 +35,17 @@ export default () => (
 
 | Property         | PropType          | Required | Default Value |
 | ---------------- | ----------------- | -------- | ------------- |
-| contentId        | string            | false    | null          |
 | buttonText       | string            | true     | -             |
+| className        | string            | false    | null          |
+| contentId        | string            | false    | null          |
 | headline         | string            | true     | -             |
 | invertVertically | bool              | true     | -             |
 | onClick          | func              | true     | -             |
-| textAlign        | oneOf(TEXT_ALIGN) | true     | -             |
-| className        | string            | false    | null          |
-| style            | object            | false    | {}            |
-| tagline          | string            | false    | null          |
 | sponsor          | object            | false    | null          |
+| style            | object            | false    | {}            |
 | subheading       | string            | false    | null          |
+| tagline          | string            | false    | null          |
+| textAlign        | oneOf(TEXT_ALIGN) | true     | -             |
```

Comment:
These should remain below the required props - I know alphabetically makes sense but we found that - listing all the required props first and then optional ones next makes it easier at a glance for consumers

Within those groupings go crazy with alphabetical

Reply:
TIL

https://github.com/Skyscanner/backpack/pull/2535/commits/d0b202e3885754c3676c70bed95308631d2c8f37